actionmailer/lib/action_mailer/mail_delivery_job.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "active_job"
+
+module ActionMailer
+  # The <tt>ActionMailer::NewDeliveryJob</tt> class is used when you
```

Comment:
It looks like this one mention of `NewDeliveryJob` stuck around even though most others were updated to `MailDeliveryJob` instead?